biolink-model.yaml

```diff
@@ -10637,10 +10668,58 @@ classes:
     is_a: association
     defining_slots:
       - subject
+      - predicate
       - object
     mixins:
       - entity to phenotypic feature association mixin
       - case to entity association mixin
+    slots:
+      - negated
+      - onset qualifier
+
+  case to disease association:
+    is_a: association
+    mixins:
+      - case to entity association mixin
+    description: "An association between a Case (patient) and a Disease"
+    slot_usage:
```

Comment:
should subject be restricted with `range: case` ?

Reply:
This is already described in `case to entity association mixin` so it should be fine